core/services/chainlink/application.go

```diff
@@ -449,6 +457,7 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 		logger:                   globalLogger,
 		closeLogger:              opts.CloseLogger,
 		secretGenerator:          opts.SecretGenerator,
+		prflr:                    prflr,
```

Comment:
Might be hard to guess what `prflr` is.  Can we rename to something more explicit like `profiler`?